content/sensu-core/2.0/reference/entities.md

```diff
@@ -4,82 +4,355 @@ description: "The entities reference guide."
+    "network": {
+      "interfaces": [
+        {
+          "name": "lo",
+          "addresses": [
+            "127.0.0.1/8",
+            "::1/128"
+          ]
+        },
+        {
+          "name": "eth0",
+          "mac": "52:54:00:20:1b:3c",
+          "addresses": [
+            "93.184.216.34/24",
+            "2606:2800:220:1:248:1893:25c8:1946/10"
+          ]
+        }
+      ]
+    },
+    "arch": "amd64"
+}
+{{</ highlight >}}
 
-### What is a proxy entity?
+last_seen    | 
+-------------|------ 
+description  | Timestamp the entity was last seen, in epoch time. 
+required     | false 
+type         | integer 
+example      | {{< highlight shell >}}"last_seen": 1522798317 {{</ highlight >}}
 
-Proxy entities (formerly known as proxy clients, "Just-in-time" or "JIT" clients) are dynamically created entities, added to the entity store if an entity does not already exist for a check result. Proxy entity registration differs from keepalive-based registration because the registration event happens while processing a check result (not a keepalive message). Sensu proxy entities allow Sensu to monitor external resources (e.g. on systems and/or devices where a sensu-agent cannot be installed, such a network switches), using the defined check ProxyEntityID to create a proxy entity for the external resource. Once created, proxy entities work much in the same way as any other Sensu entity.
 
-### Creating a proxy entity
+deregister   | 
+-------------|------ 
+description  | If the entity should be removed when it stops sending keepalive messages. 
+required     | false 
+type         | boolean 
+default      | false
+example      | {{< highlight shell >}}"deregister": false {{</ highlight >}}
 
-To create a proxy entity, a check's ProxyEntityID (formerly `source`) can be provided in the check definition or published as a check result to the TCP/UDP socket. If a check result includes a ProxyEntityID attribute, a proxy entity will be created, and the check result will be stored under the newly created entity.
+deregistration  | 
+-------------|------ 
+description  | A map containing a handler name, for use when an entity is deregistered. 
+required     | false 
+type         | [Deregistration][2] 
+example      | {{< highlight json >}}
+  "deregistration": {
+    "handler": "email-handler"
+}
+{{</ highlight >}}
 
-### Viewing
+keepalive_timeout  | 
+-------------|------ 
+description  | The time in seconds until an entity keepalive is considered stale. 
+required     | false 
+type         | integer 
+default      | 120
+example      | {{< highlight shell >}}"keepalive_timeout": 120 {{</ highlight >}}
 
-To view all the entities that are currently configured for the cluster, enter:
+organization | 
+-------------|------ 
+description  | The Sensu RBAC organization that this entity belongs to.
+required     | false 
+type         | string 
+example      | {{< highlight shell >}}
+  "organization": "default"
```

Comment:
I just realized the spaces at the beginning of this line change the indentation of the examples for the org & env:

<img width="878" alt="screen shot 2018-04-11 at 10 20 49 am" src="https://user-images.githubusercontent.com/4105580/38622887-3c71e0d2-3d72-11e8-842d-97f3c7dd275b.png">

Reply:
🤦‍♀️  I don't even know why those had linebreaks and spaces. Dang you copy pasta!